Tools/clinic/clinic.py

```diff
@@ -4864,9 +4864,11 @@ def state_modulename_name(self, line: str) -> None:
         before, equals, existing = line.rpartition('=')
         c_basename: str | None
         if equals:
-            full_name, _, c_basename = before.partition(' as ')
+            full_name, _as, c_basename = before.partition(' as ')
```

Comment:
Style, but I've tended to see a leading underscore as 'unused' vs a trailing underscore as 'reusing keyword'. Feel free to reject the suggestion though!

```suggestion
            full_name, as_, c_basename = before.partition(' as ')
```

Reply:
clinic.py uses both styles :) I have no preference here.